Doc/c-api/init.rst

```diff
@@ -7,6 +7,126 @@
 Initialization, Finalization, and Threads
 *****************************************
 
+.. _pre-init-safe:
+
+Before Python Initialization
+============================
+
+In an application embedding  Python, the :c:func:`Py_Initialize` function must
+be called before using any other Python/C API functions; with the exception of
+a few functions and flags listed below.
+
+The following functions can be safetely called before Python is initialized:
+
+* Functions:
+
+  * :c:func:`PyMem_GetAllocator`
+  * :c:func:`PyMem_SetAllocator`
+  * :c:func:`PyMem_SetupDebugHooks`
+  * :c:func:`PyObject_GetArenaAllocator`
+  * :c:func:`PyObject_SetArenaAllocator`
+  * :c:func:`Py_DecodeLocale`
+  * :c:func:`Py_GetProgramName`
+  * :c:func:`Py_SetPath`
```

Comment:
If `Py_GetProgramName` is included, shouldn't other `Py_Get*` functions be included? Py_GetPath(), Py_GetVersion(), and many others.

Reply:
Py_GetPath() must not be called before Py_Initialize().

I added other Py_GetXXX() functions.